student.rb

```diff
@@ -2,8 +2,14 @@
 
 class Student < Person
   attr_reader :classroom, :parent_permission
+  attr_accessor :name, :classroom # rubocop:todo Lint/DuplicateMethods
 
-  def initialize(age, name = 'unknown', parent_permission: true, classroom: nil)
+  def initialize(name) # rubocop:todo Lint/MissingSuper
+    @name = name
+    @classroom = nil
+  end
+
+  def initialize(age, name = 'unknown', parent_permission: true, classroom: nil) # rubocop:todo Lint/DuplicateMethods
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
   end
```

Comment:
You did a great job working on the changes requested 💯 , however, new bugs were introduced:

- [ ] Please avoid disabling linter rules as they serve as a way of having consistent code formatting and ensuring best practices. 
- [ ] Currently, you have your `:classroom` attribute repeated in your `attr_reader` and `attr_accessor` please avoid this repetition. 🤓 🔧 
- [ ] You also have two initialize methods, please keep only one! 🤓 🔧 


- [ ] Also, according to the previous review, the requirement:
> You did a great job working on this milestone! 100 However, in order to set up associations correctly, you need to implement the relationship in both ways. Here's a [refresher](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) from a previous lesson. Currently, you can't assign a classroom to a student, you can only add students to the classroom. nerd_face wrench

Hasn't been implemented completely yet. Please make sure to fix this. 💡 Hint: create a custom `classroom=` setter method to achieve this, just like in the lesson provided. 🤓